libs/runtime/unit/platform_unit_test.cpp

```diff
@@ -1225,3 +1226,79 @@ TEST_CASE("verify random", "[platform]")
     // Verify that the random number generators do not have a dominant frequency.
     REQUIRE(!has_dominant_frequency(SEQUENCE_LENGTH, ebpf_random_uint32));
 }
+
+TEST_CASE("work_queue", "[platform]")
+{
+    _test_helper test_helper;
+    test_helper.initialize();
+    struct _work_item_context
+    {
+        LIST_ENTRY list_entry;
+        KEVENT completion_event;
+    } work_item_context;
+
+    ebpf_list_initialize(&work_item_context.list_entry);
+
+    KeInitializeEvent(&work_item_context.completion_event, NotificationEvent, FALSE);
```

Comment:
do we have a cxplat api that would work here?

Reply:
Filed issues for each kernel API without equivalent in cxplat.